web/src/app/app.material-module.ts

```diff
@@ -2,14 +2,38 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { NgModule } from '@angular/core';
 import { MatCardModule } from '@angular/material/card';
 import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatIconRegistry } from '@angular/material/icon';
+import { DomSanitizer } from '@angular/platform-browser';
+
 @NgModule({
   declarations: [],
   exports: [
     BrowserAnimationsModule,
     MatCardModule,
     MatButtonModule,
+    MatIconModule,
   ],
   providers: [],
   bootstrap: [],
 })
-export class AppMaterialModule { }
+export class AppMaterialModule {
+  constructor(private matIconRegistry: MatIconRegistry, private domSanitizer: DomSanitizer) {
+    this.matIconRegistry.addSvgIcon(
+      'github',
+      this.domSanitizer.bypassSecurityTrustResourceUrl('../assets/images/Github.svg')
```

Comment:
the image names are inconsistent with some capitalised and some not. Can we make them all lowercase please

Reply:
Okay @eddiejaoude . Will do it 👍